src/feedback.tsx

```diff
@@ -242,7 +223,6 @@ export const FeedbackScreen: React.FC<FeedbackScreenProps> = ({
 
     return (
         <div style={{position: 'relative'}}>
-            {isInverse && <FeedbackScreenOverscrollColor />}
```

Comment:
shouldn't be needed because the FixedFooterLayout is already setting a fixed background that covers the whole screen

Reply:
see master vs this branch:
![image](https://github.com/Telefonica/mistica-web/assets/1849576/164b2573-df4e-49e5-a979-c8a1f0d46c84)
![image](https://github.com/Telefonica/mistica-web/assets/1849576/36429566-b528-45a6-b409-2101d66d284f)

There is also a small white line over the footer when scrolling down and a wrong top color when scrolling up:

https://github.com/Telefonica/mistica-web/assets/1849576/ffbdbd00-df17-4a3d-a361-0d8a18a07455

Reply:
ok, will restore the top overscroll